src/components/ProductSection/ProductSection.module.css

```diff
@@ -0,0 +1,54 @@
+.productSection {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 0 15px;
+
+  &.reverseClass {
```

Comment:
I wouldn't use the word `class` in the name of the class. We already know that this is a class. I would suggest something like `reversedProductSection`.